viz/src/App.vue

```diff
@@ -22,7 +85,18 @@ import TheTape from "./components/TheTape.vue";
       <main>
         <div class="mx-auto max-w-7xl sm:px-6 lg:px-8">
           <div class="px-4 py-8 space-y-10 sm:px-0">
-            <TheTape />
+            <TheTape
```

Comment:
The thing is that we could decide to change our client framework that we wouldn't even feel it at all ( at least for the logical part ) thanks to xstate